### PR TITLE
Single video page

### PR DESCRIPTION
**Pull request by chanchal16**
- Created single video page
- shows details about video
- shows a list of similar videos

https://user-images.githubusercontent.com/56486295/165034072-a50624cc-8d89-4c73-93b9-19b31ef40071.mp4

### CODE REVIEW

**Review by sonishreyas — APPROVED** [review]
Code looks good and easy to understand

File: src/styles/video.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.container{
+    padding: 1rem;
+    display: grid;
+    grid-template-columns: repeat(6,minmax(0,1fr));
+    
+}
+.video-container{
+    grid-column-start: span 4;
+    /* background-color: brown; */
+    /* margin: auto; */
```
Comment (anshtripathi01) [inline src/styles/video.css:10]: If it's not required remove it

File: src/styles/video.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+.similar-videos{
+    grid-column-start: span 2;
+    /* background-color: blanchedalmond; */
```
Comment (anshtripathi01) [inline src/styles/video.css:14]: Same here

**Review by anshtripathi01 — APPROVED** [review]
Do above changes otherwise code structure,format and UI looks good go ahead

**Pull request merged**